src/main/java/co/kr/muldum/application/teamspace/TeamPageQueryService.java

```diff
@@ -0,0 +1,36 @@
+package co.kr.muldum.application.teamspace;
+
+import co.kr.muldum.application.teamspace.dto.TeamPageQueryResponseDto;
+import co.kr.muldum.domain.teamspace.model.Team;
+import co.kr.muldum.domain.teamspace.repository.TeamRepository;
+import co.kr.muldum.domain.teamspace.repository.TeamspaceMemberRepository;
+import co.kr.muldum.global.exception.CustomException;
+import co.kr.muldum.global.exception.ErrorCode;
+import co.kr.muldum.global.util.SecurityUtil;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+public class TeamPageQueryService {
+
+    private final TeamRepository teamRepository;
+    private final TeamspaceMemberRepository teamspaceMemberRepository;
+
+    @Transactional(readOnly = true)
+    public TeamPageQueryResponseDto getTeamPage(Long teamId) {
+        Team team = teamRepository.findById(teamId)
+                .orElseThrow(() -> new CustomException(ErrorCode.NOT_FOUND));
+
+        if (!teamspaceMemberRepository.existsByTeamIdAndUserId(teamId, SecurityUtil.getCurrentUserId())) {
```

Comment:
이 부분 자체를 validate라는 메서드로 분리하여 작성하는 것은 어떨까요?
```suggestion
        validate(teamId, SecurityUtil.getCurrentUserId());
```
```
void validate(teamId, userId){
if (!teamspaceMemberRepository.existsByTeamIdAndUserId(teamId, SecurityUtil.getCurrentUserId())) {
 throw new CustomException(ErrorCode.FORBIDDEN);
        }
```